src/coreclr/inc/jithelpers.h

```diff
@@ -309,6 +309,7 @@
     JITHELPER(CORINFO_HELP_JIT_REVERSE_PINVOKE_EXIT_TRACK_TRANSITIONS,  JIT_ReversePInvokeExitTrackTransitions,  METHOD__NIL)
 
     JITHELPER(CORINFO_HELP_GVMLOOKUP_FOR_SLOT, NULL, METHOD__NIL)
+    JITHELPER(CORINFO_HELP_INTERFACELOOKUP_FOR_SLOT, NULL, METHOD__NIL)
```

Comment:
How hard would it be to add an implementation for coreclr? Currently [jit-cfg](https://dev.azure.com/dnceng-public/public/_build?definitionId=155) is likely to be broken. It would be nice to make it work without having to make the JIT side codegen different from NAOT pattern.

Alternatively I guess it would be even more ideal if we switched the testing to a CFG-enabled outerloop NAOT run now that the support is more mature... But that's probably a bit more work.

Reply:
I am not totally sure what the difference between CID and VSD is, but from the JIT codegen standpoint it looks identical to me.

In the worst case I would be ok with guarding the transformation in lowering under a NAOT check.

Reply:
@jkotas if I wanted to implement a CORINFO_HELP_INTERFACELOOKUP_FOR_SLOT on CoreCLR, would I need to somehow call `VirtualCallStubManager::ResolveWorker`? It's not exactly clear to me where I would find the parameters to call it so wondering if there's a better way.

Reply:
I have pushed a commit into this PR that shows what it may look like on win x64. It calls the slow resolve helper every time. If it is too slow for the test, it is possible to improve the perf by adding a NAOT-like fast path once David's PR https://github.com/dotnet/runtime/pull/111771/ goes in.

Reply:
> I have pushed a commit into this PR that shows what it may look like on win x64

Thank you ❤️! I'll kick off a JIT-cfg run to see if the perf is acceptable. I think we'd basically only care if this more than quadruples the time it takes for the CI to run. These legs run very infrequently.